crux-lucene/src/crux/lucene.clj

```diff
@@ -226,10 +231,7 @@
                                  (for [v (cc/vectorize-value v)
                                        :when (string? v)]
                                    [a v]))))
-            :let [id-str (->hash-str (->DocumentId e a v))
```

Comment:
we do need this, because of duplication over time - but it could be a vector so that we're not tempted to sort the map

Reply:
vectors aren't supported for IdToBuffer so I'm going with string concatenation instead